### ISSUE TITLE
Save determined asset type to asset

### ISSUE THREAD

**Issue by MurmeltierS** [unshopable/melter#2]
### What type of change do you want to see?

New feature

### Overview

Currently all assets are 'sections'. This seems pretty suboptimal.
I would see this as a potentials feature for the internal path plugin.

https://github.com/unshopable/melter/blob/9e0f78c6fc5cad65bd566958c00d1f881abc7d84/src/Compilation.ts#L74

```diff
    compiler.hooks.emitter.tap('PathsPlugin', (emitter: Emitter) => {
      emitter.hooks.beforeAssetAction.tap('PathsPlugin', (asset: Asset) => {
        const assetType = this.determineAssetType(paths, asset.source.relative);

        if (!assetType) return;
+       asset.type = assetType
        const assetSourcePathParts = asset.source.relative.split('/');

        let assetFilename: string = '';

        if (assetSourcePathParts.at(-2) === 'customers') {
          assetFilename = assetSourcePathParts.slice(-2).join('/');
        } else {
          assetFilename = assetSourcePathParts.at(-1)!;
        }

        const assetTargetPath = this.resolveAssetTargetPath(
          compiler.cwd,
          output,
          assetType,
          assetFilename,
        );

        asset.target = assetTargetPath;
      });
    });
  }

```

### Motivation

would be nice to know the asset type i guess

**Comment by MurmeltierS** [comment]
@davidthorand thoughts?

**Comment by MurmeltierS** [comment]
fixed in #3
